doc/content/modules/user-manual/pages/release-notes/2024.9.0.adoc

```diff
@@ -138,6 +138,7 @@ The source of the edge (the UseCase or Requirement) can be reconnected to anothe
 - Add support for public import in direct edit specializations.
 It is now possible to specialize an element with a qualified name containing namespaces importing the required features.
 For example, `myAttribute : ISQ::MassValue` now correctly types the attribute with `ISQBase::MassValue`.
+- Display prefix keywords in label for Definition and Usage graphical node.
```

Comment:
you forgot the final s on node
I wanted nodes, not node

Reply:
Done